coursemapper-kg/recommendation/app/services/course_materials/recommendation/youtube_service.py

```
@@ -42,17 +43,65 @@
         api_version,
         developerKey="AIzaSyClxnNwQ1x34pGioQazLlGxOjO9Fp2GGTY",
     )
+    DEVELOPER_KEYS = [  
+                        "AIzaSyBphZOn7EJmPMmZwrB71aepaA5Rbuex9MU",
+                        "AIzaSyADNntK6m7DbA6eZFYOa9Y8e6IYHykUUFE",
+                        "AIzaSyD_CGmR_Voq4DIV5okRaR6G8adoe-ZSZsM",
+                        "AIzaSyClxnNwQ1x34pGioQazLlGxOjO9Fp2GGTY",
+                        "AIzaSyB2Wck31LUlgsqI7dgTcC2dMeeVXgb9TDI"
+                    ]
+
+    def search_youtube_videos(self, developer_keys, query, top_n=50, api_service_name="youtube", api_version="v3"):
```

Ruff (0.12.2)

Missing return type annotation for public function `search_youtube_videos`

(ANN201)